docs/pages/branding/home.tsx

```diff
@@ -1,31 +1,48 @@
 import * as React from 'react';
-import { ThemeProvider } from '@material-ui/core/styles';
+import ThemeProvider from 'docs/src/modules/ThemeContext';
 import CssBaseline from '@material-ui/core/CssBaseline';
 import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
 import AppHeader from 'docs/src/layouts/AppHeader';
 import Container from '@material-ui/core/Container';
 import Hero from 'docs/src/components/home/Hero';
-import brandingTheme from 'docs/src/modules/brandingTheme';
 import References from 'docs/src/components/home/References';
+import ProductSuite from 'docs/src/components/home/ProductSuite';
+import ValueProposition from 'docs/src/components/home/ValueProposition';
+import DesignSystemComponents from 'docs/src/components/home/DesignSystemComponents';
+import Testimonials from 'docs/src/components/home/Testimonials';
+import Sponsors from 'docs/src/components/home/Sponsors';
+import HeroEnd from 'docs/src/components/home/HeroEnd';
+import AppFooter from 'docs/src/layouts/AppFooter';
 
 export default function Home() {
```

Comment:
General feedback on the responsive side of things: make the distance between sections smaller, maybe by reducing top and bottom margin/padding on each of them. On my demo, I used 20px for top/bottom padding on each section.

<img width="269" alt="Screen Shot 2021-07-29 at 18 07 32" src="https://user-images.githubusercontent.com/67129314/127566287-3561e4b4-ae87-4501-8ad3-1101933610a6.png">